visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -323,16 +323,17 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
                     this.props.accuracyPickerProps.selectedAccuracyKey);
             });
             const disparityMetric = this.state.disparityInOutcomes ?
-                (this.props.dashboardContext.modelMetadata.predictionType === PredictionTypes.binaryClassification ?
-                    "selection_rate" : "average") :
+            (this.props.dashboardContext.modelMetadata.predictionType === PredictionTypes.binaryClassification ?
```

Comment:
I think in the case of binary classifier, we actually want a different set of options available. EG, this.state.disparityInOutcomes should no longer be a bool but instead an enum for several options. The ui should show multiple options if a classifier, and set the disparity metric here based on the state of that switch.